src/js/views/patients/patient/sidebar/sidebar_views.js

```diff
@@ -137,20 +144,20 @@ const widgetView = View.extend({
     const widget = this.getOption('widget');
     const model = this.getOption('patient');
 
-    if (_.isFunction(widget)) return new widget(_.extend({ model }, this.model.get('widget_config')));
+    if (_.isFunction(widget)) return new widget(_.extend({ model }, this.model.get('definition')));
 
-    return _.extend({ model }, this.model.get('widget_config'), widget);
+    return _.extend({ model }, this.model.get('definition'), widget);
   },
 });
 
 const SidebarView = CollectionView.extend({
-  className: 'patient-sidebar flex-region',
+  className: 'patient-sidebar',
```

Comment:
Not sure why `flex-region` was on here..  racking my brain for why it was.. but I guess it's ok to remove it.